### PR TITLE
Define required YAML headers and add missing YAML header to v1 docs

### PR DESCRIPTION
**Pull request by millicentachieng**
Required YAML headers:
- title
- description
- author
- ms.prod
- localization_priority
- doc_type

`doc_type` YAML header can take values: _enumPageType_, _apiPageType_, _resourcePageType_ or _conceptualPageType_

### CODE REVIEW

**Comment by Lauragra** [comment]
@millicentachieng , FYI, the updates to the Intune files will be overwritten with each monthly drop. Do we have a plan for that? Let me know when you're ready to merge this and the beta PR.

**Comment by millicentachieng** [comment]
@Lauragra, I'll be setting up a meeting for next week with you and @angelgolfer-ms to discuss this as well as introduction of other validation rules that the docs team will need to be aware of.

File: api-reference/v1.0/api/attachment-delete.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 title: "Delete attachment"
 description: "Delete an attachment from a calendar event, mail message, or group post."
 localization_priority: Normal
+author: ""
+ms.prod: <ms.prod>
```
Comment (Lauragra) [inline api-reference/v1.0/api/attachment-delete.md]: Let's not add placeholder values for ms.prod. Can we add the correct values or leave them blank?

**Comment by Lauragra** [comment]
@millicentachieng , please run the script to add missing titles and descriptions and add that update to this PR. Thanks.

**Comment by millicentachieng** [comment]
Thank you for the reminder @Lauragra. The script will need a little bit of work but I intend to have this done by Friday this week.

**Comment by millicentachieng** [comment]
@Lauragra I have added missing titles and description in the YAML headers. This is now ready to merge.

**Comment by Lauragra** [comment]
@rolyon , can you follow up separately about updating the YAML headers in the tool that outputs the Intune files? This PR includes updates to Intune files but those updates will get overwritten. I'd like to merge this PR but let's make sure we're tracking this work item. Thanks!

**Pull request merged**